src/main/java/org/spongepowered/api/util/Color.java

```diff
@@ -83,7 +87,7 @@
      * @return The color object
      */
     public static Color ofHex(int hex) {
-        checkArgument(0 <= hex && hex <= 0xFFFFFF, "hex (" + hex + ") must be in range 0 to 0xFFFFFF");
+        checkArgument(0 <= hex && hex <= 0xFFFFFF, "hex (%s) must be in range 0 to 0xFFFFFF", hex);
```

Comment:
Since you show the decimal value in the bracets, it would be a good idea to show the decimal value for `0xFFFFFF` as well.

`"hex (%s) must be in range 0 to 0xFFFFFF (16777215)"`

Reply:
Good point; I can also include the hex value `0xFF` with `ofRgb` errors for consistency.

Would it be beneficial to include a print out of the hex value as well using `Integer.toHexString(hex)` and likewise for the `ofRgb` messages?